src/views/template/warehouse/warehouse.html

```diff
@@ -9,13 +9,13 @@
         </span>
     </div>
     <div>
-        <div class="progress no-border-radius">
+        <div class="progress no-border-radius margin-bottom-0">
```

Comment:
This puts the containers and the progress bar awfully close to each other. Also, it makes the panel-body padding look 'weird', so perhaps leave this out?